cmd/daemon/main.go

```diff
@@ -17,9 +17,24 @@ limitations under the License.
 package main
 
 import (
+	"os"
+
+	"github.com/spf13/cobra"
 	"k8s.io/klog/v2"
 )
 
+var (
+	rootCmd = &cobra.Command{
+		Use:   "wicd.exe",
+		Short: "Windows Instance Config Daemon",
+		Long: "The Windows Instance Config Daemon performs multiple functions related to maintaining the expected " +
+			"state of a Windows Node.",
+	}
+)
+
 func main() {
-	klog.Info("Daemon is running...")
```

Comment:
Will there be any log message showing WICD is running?

Reply:
@selansen yes the above help text will be printed when WICD is ran https://github.com/openshift/windows-machine-config-operator/pull/1043#issuecomment-1113515273